app/[lng]/components/ProductCard/index.js

```diff
@@ -34,14 +47,14 @@ const ProductCard = ({ product }) => {
     }
   };
 
-  const stockText = product.amount > 0 ? "В наличии" : "Нет в наличии";
+  const stockText = (product.amount > 0) ? t('availability_1') : t('availability_2');
```

Comment:
Можно сократить немного.
```suggestion
  const stockText = !!product.amount ? t('availability_1') : t('availability_2');
```